src/app/admin/components/reports/reports.directive.js

```diff
@@ -133,6 +133,13 @@
                 case 'cp-surveillance':
                 case 'cp-cap':
                 case 'cp-other':
+                case 'cp-questionable':
```

Comment:
We don't have this case any more. There is no navigation to the "questionable activities" options